app/Exceptions/Handler.php

```diff
@@ -93,6 +94,6 @@ private function fejlvarp_exception_handler($exception)
                     )))
         );
         $context = stream_context_create($opts);
-        file_get_contents('http://fejlvarp.wooinvoice.dk', false, $context);
+        $content = file_get_contents('http://fejlvarp.wooinvoice.dk', false, $context);
```

Comment:
@negoziator 
Skal man altid sørge for at tildele svaret til en variabel uanset, om man bruger den?

Reply:
Men jeg forstår faktisk ikke hvad der sker her ? 

Hvorfor læser vi overhovedet content ? 🤔

Reply:
Det kan godt være, det er mig, der har misforstået noget. Men ved at bruge file_get_contents, så bliver der implicit sendt et http request til http://fejlvarp.wooinvoice.dk
Og det er det, der er ønsket her.
Man kunne selvfølgelig gå hele vejen med Http:: osv., men jeg har haft en tanke om, at det nok er så smart lige i disse grundlæggende fejlhåndterings-funktioner at bruge teknologi som indkapsulerer alt for meget.
Og det kan godt være, det bare er mig, der ikke ved bedre.
Hvad tænker du, og giver det mening, det jeg skriver?

Reply:
Alright -  det var jeg ikke klar over 😅 

Jeg ville have troet at man så skulle bruge `file_put_contents` 

Så syntes jeg bare vi skal ignorerer den fejl 😄